devbox

```diff
@@ -1,44 +1,50 @@
 #!/bin/bash
-set -euo pipefail
-IFS=$'\n\t'
-
-if ! which ansible-playbook > /dev/null 2>&1; then
-  echo "Could not find ansible. Please install ansible first"
-  exit 1
-fi
-
-BASEDIR=$(dirname "$0")
-
-if [ ! -f "${BASEDIR}/vars.yml" ]; then
-  echo -n "mail: "
-  read MAIL
-
-  echo -n "displayName: "
-  read DISPLAY_NAME
-
-  echo -n "gpg key: "
-  read GPG_KEY
-
-  cat > "${BASEDIR}/vars.yml" <<EOF
-mail: ${MAIL}
-displayName: ${DISPLAY_NAME}
-gpgKey: "${GPG_KEY}"
-EOF
-fi
-
-if which python3 > /dev/null 2>&1; then
+set -o errexit
+set -o nounset
+set -o pipefail
+
+function ensureAnsible() {
+  ###################
+  # Install ansible #
+  if ! grep -q "ansible/ansible" /etc/apt/sources.list /etc/apt/sources.list.d/*; then
+      echo "Adding Ansible PPA"
+      sudo apt-add-repository ppa:ansible/ansible -y
+  fi
```

Comment:
Ubuntu 22.04 already provides Ansible 2.10, so you don't have to install a PPA here